### ISSUE TITLE
Error: "flag provided but not defined: -target-host"

### ISSUE THREAD

**Issue by markerdmann** [Tenderly/tenderly-cli#5]
When I try to run the Linux version of `tenderly` inside a Docker container with this command:

`./tenderly --target-host "host.docker.internal" proxy`

I get this error:

`flag provided but not defined: -target-host`

in a rapidly repeating infinite loop.

I did check that the correct port is open on that host:

```
/src # nc -vz host.docker.internal 8545
host.docker.internal (192.168.65.2:8545) open
```

If you clone my fork of the repo (https://github.com/markerdmann/tenderly-cli), you can test this easily by running `docker-compose run app sh` in the `/demo` directory and then running CLI command above.

**Comment by BogdanHabic** [comment]
Hey Markus!

So, I've cloned your repository, and the bug does indeed exist. I'm not sure why it's happening (probably we just have to bump the version of the library we use for command line args parsing).

Until we fix this, the CLI works as expected if you first put the command name, and then declare the flags:
```bash
./tenderly proxy --target-host "host.docker.internal" 
```

Closing this for now because it doesn't affect usage, other than UX, and will probably be part of `v0.3.0`.
